### PR TITLE
Relationships polish

### PR DESCRIPTION
**Pull request by kevinkace**
Before/after: http://imgur.com/a/haAoB

- update `content-state/setField()` to unset if `val` is `undefined`
- relationship name links to relationship entry

### CODE REVIEW

File: src/types/relationship.css [hunk 1]
```diff
@@ -1,27 +1,68 @@
...
+    margin: 0.6em 0 0;
+    padding-left: 1em;
+}
+
+.li {
+    display: block;
+    position: relative;
+
+    /* Chrome is dumb with <li> and children that are flexbox */
+    &:before {
```
Comment (tivac) [inline src/types/relationship.css]: Nesting is for media queries. Please break this out.

File: src/types/relationship.css [hunk 2]
```diff
@@ -1,27 +1,68 @@
...
+    }
 }
 
 .relationship {
-    composes: global(pure-g);
-    
-    align-items: center;
+    composes: hbox from flex_css;
+
+    &:hover .actions {
```
Comment (tivac) [inline src/types/relationship.css]: Nesting

File: src/types/relationship.css [hunk 3]
```diff
@@ -1,27 +1,68 @@
...
 }
 
-.name {
-    composes: global(pure-u);
+.link {
+    color: #666;
+    text-decoration: none;
+    line-height: 1.6;
+
+    &:hover {
```
Comment (tivac) [inline src/types/relationship.css]: Nesting

File: src/types/relationship.css [hunk 4]
```diff
@@ -1,27 +1,68 @@
...
+    composes: hbox from flex_css;
+
+    margin-left: 10px;
+    opacity: 0;
+}
+
+.button {
+    composes: plain from buttons_css;
+
+    &:hover .removeIcon {
```
Comment (tivac) [inline src/types/relationship.css]: Nesting

File: src/types/relationship.js [hunk 5]
```diff
@@ -131,18 +131,29 @@ export default {
...
-                        m("p", { class : css.name },
-                            ctrl.related ? ctrl.related[key].name : "Loading..."
-                        ),
-                        m("div", { class : css.actions },
-                            m("button", {
-                                class   : css.remove,
-                                onclick : ctrl.remove.bind(ctrl, key)
-                            }, "Remove")
-                        )
+                    return m("li", { class : css.li },
```
Comment (tivac) [inline src/types/relationship.js]: `{ class : css.li }`

that classname 😆

**Comment by kevinkace** [comment]
Nested selectors removed!

**Pull request merged**